mne/source_estimate.py

```diff
@@ -2769,6 +2769,9 @@ def _gen_extract_label_time_course(stcs, labels, src, mode='mean',
     elif mode == 'pca_flip':
         # get the sign-flip vector for every label
         label_flip = _get_label_flip(labels, label_vertidx, src[:2])
+    elif mode == 'pca_flip_mean':
+        # get the sign-flip vector for every label
+        label_flip = _get_label_flip(labels, label_vertidx, src[:2])
```

Comment:
new `mode` needs to be added to public-facing docstring in the appropriate places, you can `git grep pca_flip` to figure out where